README.md

```diff
@@ -18,6 +18,9 @@ By installing Java Extension Pack, the following extensions are installed:
 - [📦 Maven Project Explorer](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven)
     - Project Scaffolding
     - Custom Goals
+- [📦 Java Dependency Viewer](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency)    
+    - View Java packages, classes, and members
```

Comment:
+ projects, resources

Reply:
I think there is no need to go into this detail

Reply:
It is import to let user know we support project & jar aka the dependency.

Reply:
Done